auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -84,52 +84,16 @@
       />
 
       <!-- Add an Existing Business Dialog -->
-      <ModalDialog
-        ref="addBusinessDialog"
-        :is-persistent="true"
-        :title="dialogTitle"
-        :show-icon="false"
-        :show-actions="false"
-        max-width="675"
-        data-test-tag="add-business"
-      >
-        <template v-slot:text>
-          <p>
-            Add an existing business to your list by providing the following required pieces of information:
-          </p>
-          <v-tooltip
-            top
-            nudge-bottom="50"
-            nudge-right="125"
-            content-class="top-tooltip">
-            <template v-slot:activator="{ on, attrs }">
-              <ul class="add-business-unordered-list">
-                <li>For <strong>cooperatives</strong>, enter the incorporation number and the passcode.</li>
-                <li>For <strong>benefit companies</strong>, enter the incorporation number and the password.</li>
-                <li>For <strong>sole proprietorships and general partnerships</strong>, enter the registration
-                  number and either <span v-bind="attrs" v-on="on" activator class="underline-dotted">the name
-                  of the proprietor or a partner</span>.</li>
-              </ul>
-            </template>
-            <span class="add-business-example-name">
-              For individuals, it should be "Last Name,<br>
-              First Name MiddleName".<br>
-              E.g. Watson, John Hamish
-            </span>
-          </v-tooltip>
-
-          <AddBusinessForm
-            class="mt-6"
-            @add-success="showAddSuccessModal()"
-            @add-failed-invalid-code="showInvalidCodeModal($event)"
-            @add-failed-no-entity="showEntityNotFoundModal()"
-            @add-failed-passcode-claimed="showPasscodeClaimedModal()"
-            @add-unknown-error="showUnknownErrorModal('business')"
-            @on-cancel="cancelAddBusiness()"
-            @on-business-identifier="businessIdentifier = $event"
-          />
-        </template>
-      </ModalDialog>
+      <AddBusinessDialog
+        :dialog="addBusinessDialog"
```

Comment:
Instead of calling methods on the ref (ie, inside the ModalDialog class), now this boolean variable is used to show/hide this dialog.

Reply:
This is great